evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -504,7 +505,23 @@ def test_describe(caplog, logistic_regression_binary_pipeline_class):
     assert "Model Family: Linear" in out
```

Comment:
We should add a test to check the model family for non linear pipelines.

Reply:
There is technically one right below, line 521! I can add something more explicit as well though

Reply:
Great! Thanks for pointing that one out to me hehe